front-end/swift-buy/src/molecules/footer/index.jsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import "./footer.css";
+
+const Footer = () => {
+  return (
+    <div className="footer">
+      <div className="footer-sections">
+        <div className="footer-inner-sec">
+          <span className="footer-title">SwiftBuy</span>
+          <div className="sign-up-btn">
+            <input
+              className="footer-input"
+              type="text"
+              placeholder="Enter Email Address"
+            ></input>
+            <button className="footer-btn">Sign Up</button>
+          </div>
+          <p className="footer-content">Contact Info</p>{" "}
+          <p className="footer-content-add">
+            17 Princess Road, London, Greater London NW1 8JR, UK
+          </p>
+        </div>
+        <div className="footer-right">
```

Comment:
This whole list of different headings should be seperate genric component
as you can see its used thrice with same structure only thing which is dynamic in the data in the list